docs/UserGuide.md

```diff
@@ -3,7 +3,10 @@ layout: page
 title: User Guide
 ---
 
-Chesse Inventory Management (CHIM) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Chesse Inventory Management (CHIM) is a **desktop app for managing the contacts, inventory, and orders of a home-based
+cheese making business. The app is optimized for use via a Command Line Interface** (CLI) while still having the
```

Comment:
Sozz, i realized the bolding looks a bit weird, would be good to keep the bolding within the same sentence.

Reply:
Okay i just removed the bolding entirely, thought it was unnecessary (dk why I typed it this way, must have missed it).